### PR TITLE
Pulldown catalog values

### PR DESCRIPTION
**Pull request by whenhellfreezes**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

Helm does actually an okay job at merging a parent charts values with the value.yaml of its children. As such I am pulling values into the children charts to lessen confusion. This PR is for catalog's values.

Where greymatter.yaml differred from catalog/values.yaml I changed catalog/values.yaml. These fields are:
- client_identity
- client_email
- server_0_address
- config_polling_interval

These values are now conformant to the post svc.cluster.local -> svc change and the impersonation changes.

<br/><br/>

2. What **changes to custom.yaml** are required?

This change is backwards compatible with most custom.yaml/greymatter.yaml from the past since those over specify information. The only problem is if greymatter.yaml _didn't_ specify the four fields mentioned. Both greymatter.yaml and the di2e version **did** specify.

We could drop the changes to catalog/values.yaml to make this not an issue.

<br/><br/>

3. Have you documented any additional setup steps or configurations options?

NA

<br/><br/>

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

Tested in minikube.

<br/><br/>

### CODE REVIEW

**Comment by whenhellfreezes** [comment]
Testing was done with `make dev` do I need to update all the version numbers?

File: catalog/values.yaml [hunk 1]
```diff
@@ -63,13 +63,13 @@ catalog:
       value: 'gmdata'
     config_polling_interval:
       type: value
-      value: '300s'
+      value: '30s'
```
Comment (kaitmore) [inline catalog/values.yaml]: FYI I think this was supposed to be 300s, see convo here https://github.com/DecipherNow/helm-charts/pull/431/files#diff-4a4f56282677fc72e58c4d8cb15d1f24R66

Comment (whenhellfreezes) [reply catalog/values.yaml]: I can change it 300s again. Though given that greymatter.yaml was still at 30s it is overriding to a bad value right now on `release-2.1`.

File: catalog/values.yaml [hunk 2]
```diff
@@ -126,10 +126,10 @@ catalog:
       value: 'world'
     client_identity:
       type: value
-      value: 'CN=gm-control,OU=Engineering,O=Decipher Technology Studios,=Alexandria,=Virginia,C=US'
+      value: 'C=US,ST=Virginia,L=Alexandria,O=Decipher Technology Studios,OU=Engineering,CN=*.greymatter.svc.cluster.local'
```
Comment (chrisbsmith) [inline catalog/values.yaml]: This should really be `CN=gm-catalog,OU=Engineering,O=Decipher Technology Studios,=Alexandria,=Virginia,C=US`.  
```suggestion
      value: 'CN=gm-catalog,OU=Engineering,O=Decipher Technology Studios,=Alexandria,=Virginia,C=US'
```

File: catalog/values.yaml [hunk 3]
```diff
@@ -126,10 +126,10 @@ catalog:
...
     client_email:
       type: value
-      value: 'gm-control@deciphernow.com'
+      value: 'engineering@deciphernow.com'
```
Comment (chrisbsmith) [inline catalog/values.yaml]:
```suggestion
      value: 'gm-catalog@deciphernow.com'
```

File: catalog/values.yaml (hunk 2 above)
Comment (chrisbsmith) [reply catalog/values.yaml]: At least I believe it should, please test with this value.

Comment (whenhellfreezes) [reply catalog/values.yaml]: Will test with your suggestions. What I set it to again is what `release-2.1` is doing right now.

Comment (whenhellfreezes) [reply catalog/values.yaml]: Crashes with your suggestions. :(

Comment (whenhellfreezes) [reply catalog/values.yaml]: Seems like it can connect but then doesn't have permissions with data.

Comment (chrisbsmith) [reply catalog/values.yaml]: Please post the logs.

**Comment by whenhellfreezes** [comment]
Rolled back

**Comment by chrisbsmith** [comment]
@whenhellfreezes it may have failed b/c I improperly told you to use `gm-catalog@deciphernow.com` for the email address.  It looks like we configure the `gm-catalog` persona to use `engineering@deciphernow.com`.  Please update to try using the `gm-catalog` user again but with the engineering email.  

For reference, you can see the users we load into internal JWT below.  The data for the DN has to match the email for JWT to process it properly.

https://github.com/DecipherNow/helm-charts/blob/1a5054887114cdd32e3c7f287e8e0917d6e6f53a/greymatter.yaml#L668-L681

**Comment by whenhellfreezes** [comment]
> @whenhellfreezes it may have failed b/c I improperly told you to use `gm-catalog@deciphernow.com` for the email address. It looks like we configure the `gm-catalog` persona to use `engineering@deciphernow.com`. Please update to try using the `gm-catalog` user again but with the engineering email.
> 
> For reference, you can see the users we load into internal JWT below. The data for the DN has to match the email for JWT to process it properly.
> 
> https://github.com/DecipherNow/helm-charts/blob/1a5054887114cdd32e3c7f287e8e0917d6e6f53a/greymatter.yaml#L668-L681

Worked. Changing only DN and keeping email was successful. Set chart versions as well.

**Comment by whenhellfreezes** [comment]
Took the scenic route to fixing merge conflicts.

**Comment by whenhellfreezes** [comment]
@chrisbsmith can I get another look at this?

**Pull request closed without merging**